### ISSUE TITLE
files without extension (like makefile)

### ISSUE THREAD

**Issue by kimstik** [Maximus5/FarPlugins#13]
It seems that there is no way to set rules for this kind of files.
I am wrong?

**Comment by Maximus5** [comment]
Correct. `LoadCommentSettings` function unfortunately was designed for extensions only.

**Comment by Maximus5** [comment]
File masks would be more convenient of course. PRs are welcome ;)

**Comment by kimstik** [comment]
PR is tested on my side for about 5 years ;)
